CODEOWNERS

```diff
@@ -17,7 +17,7 @@
 /src/05-derivatives/04-structural-derivatives.md @edickie @ahoopes
```

Comment:
No related but why are we getting an error here, even though this seems to be a valid username: https://github.com/ahoopes